python/lsst/ts/ledprojector/led_controller.py

```diff
@@ -520,6 +541,10 @@ def _blocking_switch_multiple_leds(
         """
         # confirm that the lengths of both arrays match
         if len(identifiers) != len(led_states):
+            self.log.error(
+                f"Length of identifiers: {str(len(identifiers))}"
```

Comment:
you don't need to convert `len(something)` to a str here. Also, you may want to add a punctiontion and space at the end of the first line. Something like this:

```
f"Length of identifiers: {len(identifiers)}. "
f"Length of led_states: {len(led_states)}."
```